### PR TITLE
[wip] Fix duplicate logging messages

### LINKED ISSUE #3458: Model summarize displayed twice before training starts
## 🐛 Bug

Model summarize is called in two spots, which results in duplication in the logs:
- It's called once in the training loop: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/trainer/training_loop.py#L160

And I'm unsure how it's called again

119083522_2815247905426328_3914141584028152470_n

The first time is with the logger.info(...) since it's formatted. The second time it appears to be logged with `print(...)` since the table formatting doesn't handle newlines

### To Reproduce
Run any training with `weights_summary` set on the trainer

### Expected behavior
We only call summarize once

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Fixes #3458
Logs appear twice in command line. Was only able to observe this on Windows.
I don't actually understand the fix, I found it by accident. Please, if you know why this change fixes the logging, let me know.

# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by awaelchli** [comment]
@ananthsub this fixes it for me, I saw it only in windows appearing twice. 
I found this fix by accident. I actually don't understand what it is doing. I checked the fix on my linux and windows and it prints normally.

File: pytorch_lightning/__init__.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 import logging as python_logging
 
-_logger = python_logging.getLogger("lightning")
+_logger = python_logging.getLogger(__name__)
```
Comment (ananthsub) [inline pytorch_lightning/__init__.py:38]: @awaelchli  Could you explain why this fixes the DDP issue? I don't quite follow

Comment (awaelchli) [reply pytorch_lightning/__init__.py:38]: @ananthsub so the duplicate logging is not ddp related. in fact, I observed this on single gpu, and even cpu. as I said only on windows, but I suspect there is just a subtle difference between how these logging imports work between python versions and platforms. For some reason, if we don't use the module name as name for the logging instance, it creates two of them who independently log. as I said, found his fix by accident, I do not have an explanation, only empirical evidence that it fixes it on my machines.

**Comment by Borda** [comment]
@awaelchli how about this one, is it still WIP or ready to review? 🐰

**Comment by edenlightning** [comment]
@ananthsub does this work for you?

**Comment by awaelchli** [comment]
I don't see this problem anymore on my side. I don't know what solved it. It was anyway strange from the beginning. 
And the linked issue was closed, so far nobody seems to have these issues anymore.

**Pull request closed without merging**